### PR TITLE
12.0 mig base remote

### PR DESCRIPTION
**Pull request by CiaranCheng**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please check Travis status and CLA

File: base_remote/README.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (elicoidal) [inline base_remote/README.rst:1]: change to png

File: base_remote/README.rst [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========
+Remote Base
+===========
+
+This module allows to store all the connected remotes (external ip addresses) to odoo.
```
Comment (elicoidal) [inline base_remote/README.rst:9]: odoo -> Odoo

File: base_remote/README.rst [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+It should be used with other modules in order to check remote's configurations.
+
+Usage
+=====
+
+When installed, all remotes will be stored by `hostname` on `res.remote`.
+They can be viewed on `Settings / Remotes`.
+The last Ip of the remote will be stored.
```
Comment (elicoidal) [inline base_remote/README.rst:17]: Ip -> IP

File: base_remote/README.rst [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/144/11.0
```
Comment (elicoidal) [inline base_remote/README.rst:21]: 11->12

File: base_remote/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2018 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': "Remote Base",
+    'version': '12.0.1.0.2',
```
Comment (elicoidal) [inline base_remote/__manifest__.py:6]: 12.0.1.0.0 (you restart the number when jumping major version to 12.0)

File: base_remote/models/res_remote.py [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+            'ip': addr,
+            'in_network': bool(hostname),
+        }
+
+    @api.model
+    def _get_remote(self, addr):
+        try:
+            hostname, alias, ips = socket.gethostbyaddr(addr)
+        except socket.herror:
+            logging.warning('Remote with ip %s could not be found' % addr)
```
Comment (elicoidal) [inline base_remote/models/res_remote.py:41]: IP

File: base_remote/views/res_remote_views.xml [hunk 7]
```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="res_remote_form" model="ir.ui.view">
```
Comment (elicoidal) [inline base_remote/views/res_remote_views.xml:3]: add copyright

**Review by elicoidal — CHANGES_REQUESTED** [review]
mostly details on my side + travis

**Comment by jarroyomorales** [comment]
@CiaranCheng @pedrobaeza Is someone working on this? Should I create a new PR to finish the migration?

**Comment by pedrobaeza** [comment]
The author should answer, but it seems not.

**Pull request closed without merging**